app/src/main/AndroidManifest.xml

```diff
@@ -201,10 +201,6 @@
                 android:value="de.danoeh.antennapod.activity.PreferenceActivity"/>
         </activity>
 
-        <meta-data
-            android:name="android.support.PARENT_ACTIVITY"
-            android:value="de.danoeh.antennapod.activity.PreferenceActivity"/>
-
```

Comment:
As far as I know, android.support.PARENT_ACTIVITY has no effect inside the application tag.